pkgs/top-level/python-packages.nix

```diff
@@ -773,13 +773,18 @@ in modules // {
   };
 
   application = buildPythonPackage rec {
-    name = "python-application-${version}";
-    version = "1.5.0";
+    pname = "python-application";
+    name = "${pname}-${version}";
+    version = "2.0.2";
+    disabled = isPy3k;
```

Comment:
Okay, reasons like this are best included in the code whenever disabling packages or tests.